ShoppIt/src/main/controllers/MainController.java

```diff
@@ -30,7 +30,7 @@ public class MainController implements Initializable {
     public void initialize(URL arg0, ResourceBundle arg1) {
         DatabaseManager DB = new DatabaseManager();
 
-        lists = DB.getFromDatabase(ShoppingList.class, "FROM ShoppingList s");
+        lists = DB.getTable(ShoppingList.class, "ShoppingList");
```

Comment:
> You mentioned that you made more specific query methods but queries still get the entire table rather than narrowing down. Is there a reason you didn't

Good question! We noticed that in the cases where the queries fetched the whole table, the functionality actually required access to each item in the table (such as in the above code snippet).

"Narrowing down" in this case meant instead of needing to write the query yourself to get an attribute or field, we added more specific methods that sets up the JPQL query for you; I should have clarified this better.